drivers/mei/README.md

```diff
@@ -0,0 +1,49 @@
+# mei extension
+
+## Installation
+
+See [Installing Extensions](https://github.com/siderolabs/extensions#installing-extensions).
+
+## Usage
+
+Enable the following modules in Talos machine config to allow access to Intel Arc discrete GPU functionality:
+- mei_hdcp
+- mei-gsc
+- mei-me
+- mei-txe
+- mei
+- mei_pxp
+- mei_wdt
+
+Enable the `mei` module in Talos machine config.
+If you need Thunderbolt/USB4 networking,
+enable `thunderbolt_net` module as well.
+
+```yaml
+machine:
+	kernel:
+		modules:
+			- name: mei_hdcp
+			- name: mei-gsc
+			- name: mei-me
```

Comment:
seems these drivers have pci id's so i guess udev would automatically load them? :thinking:

Reply:
For the use case of Intel Arc, no further changes/configs were required, including dev; assuming the modules are available, the GPU is usable.

Reply:
i meant there's no need to explicitly load them, since these modules have pci id's, it should get automatically loaded

Reply:
Yeah, same page, different words :)

Reply:
Oh, I misunderstood.

Let me run some tests and get back to you on that.

Reply:
You are correct. Docs updated.